worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/ChunkStreamManager.java

```diff
@@ -97,22 +99,20 @@ public void chunkBeingSent(long streamId) {
     StreamState streamState = streams.get(streamId);
     if (streamState != null) {
       streamState.chunksBeingTransferred++;
+      chunksBeingTransferredNum.incrementAndGet();
```

Comment:
The current logic is referred to Spark's implementation. I'm not sure how much it will benefit with this change.